### PR TITLE
Updated the documentation so it can be used as an open source package.

### PR DESCRIPTION
**Pull request by zero2one**
Make the package open source.

## Types of changes

- [x] New feature (non-breaking change which adds functionality).
- [x] Documentation update.

## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have updated the CHANGELOG accordingly.
- [x] I have read the **CONTRIBUTING** document.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,7 +1,9 @@
 language: php
 php:
-  - 7.2
   - 5.6
```
Comment (MPParsley) [inline .travis.yml]: 5.6 is no longer supported.
```suggestion
```

File: README.md [hunk 2]
```diff
@@ -1,29 +1,39 @@
...
+This package contains the following interfaces:
+
+* `ValueInterface` : the basic interface for all value objects.
+* `ValueFromArrayInterface` : extra interface that adds, to a value object, the
+  static method to create it from an array of data.
+* `ValueToArrayInterface` : extra interface that adds, to a value object, the
+  method to extract its data as an array.
+* `CollectonInterface` : the basic interface for a collection of values of the
+  same type.
+
```
Comment (MPParsley) [inline README.md]:
```suggestion
 
```

File: README.md [hunk 3]
```diff
@@ -1,29 +1,39 @@
...
+* `ValueInterface` : the basic interface for all value objects.
+* `ValueFromArrayInterface` : extra interface that adds, to a value object, the
+  static method to create it from an array of data.
+* `ValueToArrayInterface` : extra interface that adds, to a value object, the
+  method to extract its data as an array.
+* `CollectonInterface` : the basic interface for a collection of values of the
+  same type.
+  
+And the following abstract implementations:
```
Comment (MPParsley) [inline README.md]:
```suggestion
And the following abstract implementations:
```

Comment (MPParsley) [inline README.md]:
```suggestion
And the following abstract implementations:
```

File: README.md [hunk 4]
```diff
@@ -1,29 +1,39 @@
...
+* `ValueInterface` : the basic interface for all value objects.
+* `ValueFromArrayInterface` : extra interface that adds, to a value object, the
+  static method to create it from an array of data.
+* `ValueToArrayInterface` : extra interface that adds, to a value object, the
+  method to extract its data as an array.
+* `CollectonInterface` : the basic interface for a collection of values of the
+  same type.
+  
+And the following abstract implementations:   
+
```
Comment (MPParsley) [inline README.md]:
```suggestion

```

File: .travis.yml (hunk 1 above)
Comment (zero2one) [reply .travis.yml]: 24 more days...
http://php.net/supported-versions.php

**Pull request merged**